src/Tree/index.tsx

```diff
@@ -377,6 +383,43 @@ class Tree extends React.Component<TreeProps, TreeState> {
     }
   };
 
+  /**
+   * Takes a hierarchy point node and centers the node on the screen
+   * if the dimensions parameter is passed to the tree.
+   *
+   * This code is adapted from Rob Schmuecker's centerNode method.
+   * Link: http://www.robschmuecker.com/d3-js-drag-and-drop-zoomable-tree/
```

Comment:
This site is timing out for me with a `524` error :( keen to have a look at this when it comes back online 👍

Reply:
You should also be able to see the code on this site: http://bl.ocks.org/robschmuecker/7880033

The function was called `centerNode`